particles/PipeApps2/canonical.recipes

```diff
@@ -0,0 +1 @@
+import 'PipeApps.recipes'
```

Comment:
Same -- intentional? Why is `canonical.recipes` blank currently? @yuangu-google

Reply:
uncommented (and got rid of canonical.recipes - Scott says it is redundant)